src/osal/rt-kernel/osal.c

```diff
@@ -121,24 +120,27 @@ int os_snprintf(char * str, size_t size, const char * fmt, ...)
    return ret;
 }
 
-int os_save_blob(
-   int                     file_index,
```

Comment:
Can we implement these functions for rt-kernel also? They could be mostly identical to the linux version, although I would recommend unbuffered I/O (open/read/write/close), since buffered I/O is more costly in terms of resources. It wouldn't be necessary if the magic/version was a precomputed byte string.

Reply:
Implemented also for rt-kernel